Automattic/wp-calypso
# Nav Unification: introduce Classic Dark as the new default color scheme

#### Changes proposed in this Pull Request

This PR
* introduces Classic Dark as the new default color scheme
* adds Classic Bright as a copy of the previous default color scheme
* adds missing colors to Classic Blue that were previously inherited from Classic Bright default

Context: 

With the nav unification project we're introducing unified default colors across Calypso and wp-admin. @rickybanister suggested we introduce these colors to Calypso users early via a new default color scheme. The colors in this PR are based on the most recent Navigation Unification, Design Update (see p2-pbAPfg-13V).

Related issue: https://github.com/Automattic/wp-calypso/issues/47135

![new-default-color-scheme](https://user-images.githubusercontent.com/1562646/98258262-f1ec4400-1f80-11eb-982d-8c41b332825f.gif)


#### Testing instructions



* Check out branch locally, `yarn && yarn start`, visit http://calypso.localhost:3000/me/account
* OR use the [calypso.live link](https://calypso.live/?branch=feature/new-default-color-scheme) (easier!)
* Test on a user account that has never had a color scheme set. On this account the new Classic Dark color scheme should show as default now.
* Navigate around Calypso and see if everything looks as expected
* Change color schemes to Classic Bright and Classic Blue and see if those still match what you see in production.


Fixes https://github.com/Automattic/wp-calypso/issues/47135
